### ISSUE TITLE
Information about other player's controlState

### ISSUE THREAD

**Issue by derjp** [PrismarineJS/mineflayer#727]
Is there a way to check whether a player is sneaking? I couldn't find anything in the api.

**Comment by rom1504** [comment]
No.
I don't know how that get sent by the server. Try to run the bot in debug mode (https://github.com/PrismarineJS/node-minecraft-protocol#debugging)  and check what packet could indicate this

**Comment by derjp** [comment]
I checked the API once again and found the "entityCrouch" and "entityUncrouch" events: https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#entitycrouch-entity

Don't know how I could have missed them before opening this issue.

Anyways, unfortunately they don't emit any events (maybe because it's a paper server (non-vanilla)). Next I tried the "entityUpdate" event which indeed worked. You just have to check for the metadata property :)
